Foundation/NSURLSession/URLSessionTask.swift

```diff
@@ -521,7 +521,7 @@ open class URLSessionStreamTask : URLSessionTask {
 }
 
 /* Key in the userInfo dictionary of an NSError received during a failed download. */
-public let URLSessionDownloadTaskResumeData: String = "NSURLSessionDownloadTaskResumeData"
+public let URLSessionDownloadTaskResumeData: String = "URLSessionDownloadTaskResumeData"
```

Comment:
This should be kept as is for compatibility with Darwin. Also related to [SR-3403](https://bugs.swift.org/browse/SR-3403).